api/src/main/java/org/loomdev/api/event/server/connection/StatusPingEvent.java

```diff
@@ -0,0 +1,44 @@
+package org.loomdev.api.event.server.connection;
+
+import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.NotNull;
+import org.loomdev.api.event.Cancelable;
+import org.loomdev.api.event.Event;
+
+import java.net.InetAddress;
+
+public interface StatusPingEvent extends Event, Cancelable {
```

Comment:
What does Cancelable add here? (feature wise)

Reply:
Currently it forces the server to ignore the ping (doesn't send a response).

Reply:
that's fine 😄